en/installation_guide/install_in_symfony_project.rst

```diff
@@ -0,0 +1,580 @@
+Install OpenOrchestra in an existing Symfony project
+====================================================
+
+Open Orchestra is composed of three projects:
+
+- *open-orchestra* which is the CMS Back Office;
+- *open-orchestra-front-demo* which is the Front Office part that will display the sites and pages
+  created in the CMS Back Office;
+- *open-orchestra-media-demo* which will display the images loaded in the CMS Back Office.
+
+Each projects is composed to differents bundles,
+these bundle can be used in an existent Symfony application.
+
+Before use a bundle of Open Orchestra you should check if your server respect
+the `requirement <../hosting_guide/requirements.html>`_. Also it need some configurations (virtual host,
+cron, ...).
+
+This documentation explain the steps (composer, configuration, ...) to add Open Orchestra bundles
+in your application.
+
+Back Office
```

Comment:
You're introducing the explanation of a bundle inclusion and instead you explain how to add the Admin functionnality. I think there's something to rewrite somewhere to be coherent